scripts/mlp/centroidal/geometric.py

```diff
@@ -39,18 +39,18 @@ def computePhaseDuration(cs,pid):
         if duration_feet > 0.:
             duration_feet = math.ceil(duration_feet/cfg.SOLVER_DT)*cfg.SOLVER_DT
         if VERBOSE :
-            print "for phase : ",pid
-            print "dist_feet            : ",dist_feet
-            print "duration translation : ",duration_feet_trans
-            print "rot_feet             : ",rot_feet
-            print "duration rotation    : ",duration_feet_rot
-            print "duration complete    : ",duration_feet
+            print("for phase : ",pid)
+            print("dist_feet            : ",dist_feet)
```

Comment:
In python 2, you can write: `print a, b`
In python 3, you can write: `print(a, b)`
But if you write `print(a, b)` and try to use that in python2, it will print a tuple.

Workarounds are:
- say that you don't support python 2 anymore
- or add a `from __future__ import print_function` before anything else in this file

Reply:
Btw, pep8 says that it's nice to put a space after a comma ;)

Reply:
Also, you can just ignore that, and print a tuple. This is a bit uglier, that's all..

Reply:
I think that we can drop the python2 support. Anyway if the only issue of running this in python2 is the extra [ ] around the print it not enough to motivate adding extra code. 

I will format the comma as you said.

Reply:
For the format, you can pretty much just run `yapf -i <files>` (`-r` to run it recursively on directories)